client/my-sites/checkout/composite-checkout/components/wp-order-review-line-items.tsx

```diff
@@ -638,6 +638,79 @@ function FirstTermDiscountCallout( {
 	return null;
 }
 
+function IntroductoryOfferCallout( {
+	product,
+}: {
+	product: ResponseCartProduct;
+} ): JSX.Element | null {
+	const translate = useTranslate();
+	let text = String( translate( 'Discount for first period' ) );
+	if ( ! product.introductory_offer_terms?.enabled ) {
+		return null;
+	}
+
+	const intervalUnit = product.introductory_offer_terms.interval_unit;
+	const intervalCount = product.introductory_offer_terms.interval_count;
+
+	if ( product.item_subtotal_integer === 0 ) {
+		if ( intervalUnit === 'month' ) {
+			if ( intervalCount === 1 ) {
```

Comment:
Shouldn't this use the three-argument form of `translate()`? The same applies to all the locations below.
```suggestion
			text = translate(
				'First month free',
				'First %(numberOfMonths)s months free',
				{
					args: {
						numberOfMonths: intervalCount,
					},
					count: intervalCount,
				}
			);
```

Reply:
I did that at first but I got a warning telling me that I **had** to use the placeholders in both strings, that's why I ended up separating them using an if.

![image](https://user-images.githubusercontent.com/15204776/111500233-5fffff00-8709-11eb-9347-b20a8aa89c13.png)